src/compiler/diagnosticMessages.json

```diff
@@ -4132,6 +4132,14 @@
         "category": "Error",
         "code": 4124
     },
+    "Each declaration of '{0}.{1}' differs in its value, where '{2}' was expected but '{3}' was given.": {
+        "category": "Error",
+        "code": 4125
+    },
+    "Values of '{0}.{1}' are not sufficiently known.": {

```

Comment:
Does it make sense to explicitly say the values are not sufficiently known/specified and therefore can't be determined to be the same?
Just wondering if everyone will read this message already knowing that's what the type checker is doing: trying to compare values to check they're the same.